test/uk/gov/hmrc/eusubsidycompliancefrontend/forms/ClaimDateFormProviderSpec.scala

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.forms
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import play.api.data.FormError
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.DateFormValues
+import uk.gov.hmrc.eusubsidycompliancefrontend.util.TimeProvider
+
+import java.time.{LocalDate, ZoneId}
+
+class ClaimDateFormProviderSpec extends AnyWordSpecLike with Matchers {
+
+  private val day = 1
+  private val month = 1
+  private val year = 2022
+
+  private val fakeTimeProvider: TimeProvider = new TimeProvider {
```

Comment:
You can make it def and we can test it using various dates? like 1 jan , 4 april, 5 april. what do you think?

Reply:
Yeah I can pull this out so other tests can use it. Will do that now.

Reply:
Pushed an update for that now.